app/models/validations/household_validations.rb

```diff
@@ -131,11 +131,17 @@ def validate_person_age_matches_economic_status(record, person_num)
     economic_status = record.public_send("ecstat#{person_num}")
     return unless age && economic_status
 
-    if age > 70 && economic_status != 4
+    if age > 70 && economic_status != 5
       record.errors.add "ecstat#{person_num}", I18n.t("validations.household.ecstat.retired_over_70", person_num:)
+      record.errors.add "age#{person_num}", I18n.t("validations.household.age.retired_over_70", person_num:)
     end
-    if age < 16 && economic_status != 8
+    if age < 16 && economic_status != 9
       record.errors.add "ecstat#{person_num}", I18n.t("validations.household.ecstat.child_under_16", person_num:)
+      record.errors.add "age#{person_num}", I18n.t("validations.household.age.child_under_16", person_num:)
+    end
+    if economic_status == 9 && age > 16
+      record.errors.add "ecstat#{person_num}", I18n.t("validations.household.ecstat.child_over_16", person_num:)
+      record.errors.add "age#{person_num}", I18n.t("validations.household.age.child_over_16", person_num:)
```

Comment:
I wonder if we could have some helper methods here to clarify what the economic status integers actually mean

Reply:
Right you are